src/System.IO.IsolatedStorage/tests/System/IO/IsolatedStorage/GetFileNamesTests.cs

```diff
@@ -57,7 +57,7 @@ public void GetFileNames_RaisesInvalidPath_Core()
             // We are no longer as aggressive with filters for enumerating files
```

Comment:
This comment isn't appropriate any more. `\\ We only throw with embedded null characters in filters`